.trivyignore

```diff
@@ -0,0 +1,5 @@
+# org.scala-lang:scala-library (from upstream opensource dependency)
+CVE-2022-36944 exp:2023-09-30
+
+# org.xerial.snappy:snappy-java (from upstream opensource dependency)
```

Comment:
This one we can definitely fix, perhaps use the kafka bom or just constraint here directly

Reply:
This is kafka server image. Not kafka-streams application. We don't have any codebase and don't build the code. We just use apache tarball distro to build our image.